### DISCUSSION TITLE
Molecular orbitals plot

### DISCUSSION THREAD

**Discussion by gfl72** [tblite/tblite#203] [General]
Is there a way to output the molecular orbitals in some useful format for plotting?

**Comment by gfl72** [comment]
If this is not possible,  is there any readily available code which can convert the tblite parameters from TOML format to xtb format?
I have modified some parameters and I can not find a way to plot the orbitals. If  the new parameters can be easily transferred to xtb code, then the MO plotting is easy since xtb can export molden file.

In any case...
Thank you very much for tblite and all your efforts to improve it!!

**Reply by marcelmbn** [reply]
Maybe @haneug can help as you have been working on a small tool connected to the same problem?

**Reply by gfl72** [reply]
Thank you very much for your response!

**Comment by awvwgk** [comment]
We now have support for writing molden and trexio files in tblite, that should be helpful for visualizing orbitals.
